glusterd2/xlator/validation.go

```diff
@@ -78,6 +78,22 @@ func validateQuota(v *volume.Volinfo, key string, value string) error {
 	return nil
 }
 
+func validateDht(v *volume.Volinfo, key string, value string) error {
+       var err error
+       if strings.Contains(key, "readdirplus-for-dir") {
```

Comment:
I can't find this key `readdirplus-for-dir` in glusterfs sources.

Reply:
Yes, the patch is under review and adding this here was a review comment: https://review.gluster.org/#/c/19559/